.github/workflows/ci.yml

```diff
@@ -454,6 +454,9 @@ jobs:
           - config: default
             suite: suite-1
             jdk: zulu_17
+          - config: hdp3
```

Comment:
I would just keep `hdp3` for to keep CI queue small. 
@findepi WDYT?

Reply:
yep, replace default with hdp3

Reply:
Replaced